automation/taskcluster/schedule_nightly_graph.py

```diff
@@ -0,0 +1,96 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import argparse
+import datetime
+import jsone
+import os
+import slugid
+import taskcluster
+import yaml
+
+from git import Repo
+from lib.tasks import schedule_task
+
+ROOT = os.path.join(os.path.dirname(__file__), '../..')
+
+
+def calculate_git_references(root):
+    repo = Repo(root)
+    remote = repo.remote()
+    branch = repo.head.reference
+
+    assert remote.url.startswith('https://github.com'), 'expected remote to be a GitHub repository (accessed via HTTPs)'
```

Comment:
Nit: raise an exception instead. An assert statement may be ignored.

Out of curiosity, what protection does this check provide?

Reply:
Riiight, the `assert` vs `exception` we talked about earlier :+1: thanks!
This protects against an `ssh` repository being used. With an SSH repository, we can't do [`$repo/raw/$commit/.taskcluster.yml`](https://github.com/mitchhentges/reference-browser/blob/b01877b34ff44c89912ce846978f47dc9b269609/automation/taskcluster/decision_task_nightly.py#L25)